pdns/recursordist/meson.build

```diff
@@ -582,3 +582,99 @@ if python.found()
     ] + man_pages,
   )
 endif
+
+if dep_systemd.found()
+  systemd_service_conf = configuration_data()
+  systemd_service_conf.set('BinDir', get_option('prefix') / get_option('bindir'))
```

Comment:
Seems unused?

Reply:
yes, but it might be that `rec_control` will be used at one point, so I left it in